docs/questions/query-builder/expressions/converttimezone.md

```diff
@@ -0,0 +1,228 @@
+---
+title: ConvertTimezone
+---
+
+# ConvertTimezone
+
+`convertTimezone` shifts a timestamp into a specified time zone by adding or subtracting the right interval from the timestamp.
+
+| Syntax                                                                | Example                                                                           |
+|-----------------------------------------------------------------------|-----------------------------------------------------------------------------------|
+| `convertTimezone(column, target, source)`                             | `convertTimezone("2022-12-28T12:00:00", "PST", "EST")`                            |
+| Shifts a timestamp from the source time zone to the target time zone. | Returns the value `2022-12-28T09:00:00`, displayed as `December 28, 2022 9:00 AM` |
+
+Timestamps and time zones are rather nasty to work with (it's easy to make mistakes, and difficult to catch them), so you should only try to use `convertTimezone` if the interpretation of your data is very sensitive to time-based cutoffs.
+
+For example, if you're tracking user logins over time, you probably won't run your business differently if some logins get counted on Mondays instead of Tuesdays. However, if you're using Metabase to do something precise, like your taxes, you (and the government) will probably care a lot more about the difference between transactions that occurred on Dec 31 vs. Jan 1.
+
+## Parameters
+
+`column` can be any of:
+- The name of a timestamp column,
+- a custom expression that returns a [timestamp](#accepted-data-types), or
+- a string in the format `"YYYY-MM-DD` or `"YYYY-MM-DDTHH:MM:SS"`.
+
+`target`:
+- The name of the time zone you want to assign to your column.
+- Time zone names are database-dependent (such as "Canada/Eastern" vs. "EST").
```

Comment:
To be precise but we use the `tz database`.https://en.wikipedia.org/wiki/List_of_tz_database_time_zones

This is what most DBs use, only sqlserver accepts WindowTimezone which is a bit different (`Pacific Standard Time` vs `PST`)

Do you think we should have this information included?